### PR TITLE
Grafana-agent: Added agent and agent-operator

### PR DESCRIPTION
**Pull request by jaxels10**
**Description of your changes:**
I have added the grafana agent operator proxy chart
I have created the grafana agent chart with the necessary resources
I have added both new charts to ct-lint and ct-install

Checklist:

* [ ] I have bumped the chart version
* [ ] I have documented my changes if this is needed
* [x] I have described my changes in the "description of your changes" field
* [x] I have tested this change on a running cluster and the Argocd dashboard shows healthy and synced
* [ ] I have created the necessary tests in the chart, if they are possible/necessary
* [x] I have squashed commits if necessary

### CODE REVIEW

File: charts/grafana-agent/chart/templates/pod-logs.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+metadata:
+  labels:
+    instance: primary
+  name: kubernetes-pods
+spec:
+  pipelineStages:
+    - docker: {}
+  namespaceSelector:
+    matchNames:
+      - {{ .Release.Namespace }}
```
Comment (MadsHT) [inline charts/grafana-agent/chart/templates/pod-logs.yaml:12]: Do we want this to only be on the this charts release namespace?
Do we not want them to be on all namespaces?

**Review by MadsHT — COMMENTED** [review]
Singular comment

File: charts/grafana-agent/chart/templates/pod-logs.yaml (hunk 1 above)
Comment (jaxels10) [reply charts/grafana-agent/chart/templates/pod-logs.yaml]: This was an error and I have now removed it.

**Review by MadsHT — DISMISSED** [review]
LGTM

**Review by MadsHT — APPROVED** [review]
LGTM

**Review by MadsHT — APPROVED** [review]
LGTM

**Review by jacobcrawford — APPROVED** [review]
LGTM

**Pull request merged**